packages/icons/src/icon/Icon.css

```diff
@@ -1,6 +1,5 @@
 .uitkIcon {
   --icon-color: var(--uitkIcon-color, var(--uitk-text-secondary-foreground));
-  --icon-size: var(--uitkIcon-size, var(--uitk-size-graphic-small));
```

Comment:
This breaks the icon size when Icon doesn't have a named size value. This is why Button stories are broken